decode_morse.rb

```diff
@@ -0,0 +1,31 @@
+# decode_morse.rb
+# ruby "/Users/andrea/Desktop/Microverse/Module 4/morse-code/decode_morse.rb"
+
+def decode_a_char(char)
+  decodemessage = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+                    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K',
+                    '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q',
+                    '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W',
+                    '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z' }
+  decodemessage[char]
+end
+
+def decode_word(word)
+  decoded_string = word.split
+  decoded_word = ''
+  decoded_string.each do |n|
+    decoded_word += decode_a_char(n)
+  end
+  decoded_word
+end
+
+def decode_message(dec_word)
```

Comment:
- [ ] While testing your app, find a bug where if I print the decoded message, I'm getting an empty space at the beginning of the message. Kindly fix this blank space. You can check it running a decode message with `p` instead of `puts`

![image](https://github.com/kerlos-bahaa/Morse-code/assets/93630700/89c902de-d43e-41d0-a79d-f2c34dfa2222)